docker/generate_makefile.py

```diff
@@ -56,16 +59,28 @@ def _print_makefile_run_template(image):
 \t-e FUZZ_TARGET=$({benchmark}-fuzz-target) \\\
 '.format(fuzzer=fuzzer, benchmark=benchmark))
 
+        params = ""
         if run_type == 'test-run':
             print('\t-e MAX_TOTAL_TIME=20 \\\n\t-e SNAPSHOT_PERIOD=10 \\')
         if run_type == 'debug':
             print('\t--entrypoint "/bin/bash" \\\n\t-it ', end='')
+        elif run_type == 'repro-bugs':
+            print('\t-v {path}:/testcases \\\n\
+\t--entrypoint /bin/bash '.format(path=bench_path),
+                  end='')
+            params = " -c \"for f in /testcases/*; do\
+                    echo _________________________________________;\
+                    echo \\$$f:;\
+                    \\$$OUT/\\$$FUZZ_TARGET -timeout=25\
+                    -rss_limit_mb=2560 \\$$f; done;\""
+
         elif run_type == 'run':
             print('\t-it ', end='')
         else:
             print('\t', end='')
 
-        print(os.path.join(BASE_TAG, image['tag']))
+        print(os.path.join(BASE_TAG, image['tag']), end='')
+        print(params)
```

Comment:
can you do a 
if (params) and only print when it is non-empty ? that would simplify tests.